lib/orglogin.dart

```diff
@@ -0,0 +1,258 @@
+        ),
+        centerTitle: true,
+        titleTextStyle: TextStyle(
+          color: Colors.black,
+        ),
+        elevation: 0,
+        backgroundColor: Color.fromARGB(0, 5, 0, 0),
+      ),
+      body: Stack(
+        children: [
+          Container(
+            decoration: const BoxDecoration(
+              // image: DecorationImage(
+              //     image: AssetImage('lib/assets/Vectorprofilepic.png')),
+              gradient: LinearGradient(colors: [
+                Color.fromARGB(255, 101, 167, 221),
+                Colors.white,
+              ], stops: [
+                0.1,
+                1.00
+              ], begin: Alignment.topCenter, end: Alignment.center),
+            ),
+            padding:
+                const EdgeInsets.only(right: 32.99, top: 151, left: 187.01),
+            child: Column(
+              children: [
+                TextField(
+                  decoration: InputDecoration(
+                      fillColor: Colors.cyan.shade100,
+                      filled: true,
+                      hintText: 'Upload Profile Picture',
+                      border: OutlineInputBorder(
+                          borderRadius: BorderRadius.circular(10))),
+                ),
+                const SizedBox(
+                  height: 40,
+                ),
+                // Row(
+                //   mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                //   children: [
+                //     Expanded(
+                //         child: ListView(
+                //       children: [
+                //         TextField(
+                //           decoration: InputDecoration(
+                //               fillColor: Colors.cyan.shade100,
+                //               filled: true,
+                //               hintText: 'From Gallery',
+                //               border: OutlineInputBorder(
+                //                   borderRadius: BorderRadius.circular(10))),
+                //         ),
+                //         TextField(
+                //           decoration: InputDecoration(
+                //               fillColor: Colors.cyan.shade100,
+                //               filled: true,
+                //               hintText: 'Take Now',
+                //               border: OutlineInputBorder(
+                //                   borderRadius: BorderRadius.circular(10))),
+                //         ),
+                //       ],
+                //     ))
+                //   ],
+                // ),
+              ],
+            ),
+          ),
+          Container(
+            padding: const EdgeInsets.only(right: 32, top: 265, left: 200),
+            child: Text(
+              'Enter Organisation Details',
+              style:
+                  TextStyle(fontWeight: FontWeight.bold, color: Colors.black),
+            ),
+          ),
+          SingleChildScrollView(
```

Comment:
Here, Container should be a parent, and SingleChildScrollVIew should be its child. Also, in order to make it scroll, specify the height of the Container.